### PR TITLE
Fix dependabot config example

### LINKED ISSUE #162: Example dependabot config is invalid
See this issue that dependabot creates automatically on a repo https://github.com/humanmade/dormakaba/issues/385

### PR DESCRIPTION
**Pull request by roborourke**
Fixes #162

### CODE REVIEW

File: other-docs/guides/automatic-updates.md [hunk 1]
```diff
@@ -17,12 +17,11 @@ updates:
   - package-ecosystem: "composer"
     # Look for `composer.json` and `composer.lock` files in the `root` directory
     directory: "/"
-    # Create pull requests as soon as updates are made available
-    schedule:
```
Comment (jazzsequence) [inline other-docs/guides/automatic-updates.md:21]: is this `schedule` line intentionally removed?

**Review by jazzsequence — CHANGES_REQUESTED** [review]
just one thing jumped out at me, looks fine otherwise.

File: other-docs/guides/automatic-updates.md (hunk 1 above)
Comment (roborourke) [reply other-docs/guides/automatic-updates.md:21]: Nope, not sure how I did that 😅

Comment (roborourke) [reply other-docs/guides/automatic-updates.md:21]: Fixed!

**Review by jazzsequence — APPROVED** [review]

**Pull request merged**